alternator/executor.cc

```diff
@@ -3145,19 +3145,58 @@ future<executor::request_return_type> executor::batch_get_item(client_state& cli
     // try again. Note that simply a missing key is *not* an error (we already
```

Comment:
Github doesn't let me put a comment on earlier lines, but note that a few lines up this comment is a FIXME about UnprocessedKeys, so now that you did this, you should remove the FIXME (but not the entire comment).

Reply:
I think you missed this comment